library/package.json

```diff
@@ -34,26 +34,27 @@
   "types": "lib/types/index.d.ts",
   "files": [
     "/lib",
-    "/bundles",
+    "/browser",
     "/styles",
     "./README.md",
     "./LICENSE"
   ],
   "sideEffects": false,
   "scripts": {
     "start": "tsc -p tsconfig.esm.json --watch",
-    "build:dev": "npm run build:esm && npm run build:styles",
-    "build:prod": "npm run build:esm && npm run build:cjs && npm run build:umd && npm run build:types && npm run build:styles && npm run build:styles:prod",
+    "build:dev": "npm run build:esm && npm run build:types && npm run build:styles",
+    "build:prod": "npm run build:esm && npm run build:cjs && npm run build:umd && npm run build:types && npm run build:styles",
     "build:esm": "tsc -p tsconfig.esm.json",
     "build:cjs": "tsc -p tsconfig.cjs.json",
     "build:umd": "webpack --colors",
     "build:types": "tsc -p tsconfig.types.json",
-    "build:styles": "postcss src/styles/default.css -o styles/default.css",
-    "build:styles:prod": "cross-env NODE_ENV=production postcss src/styles/default.css -o styles/default.min.css",
+    "build:styles": "npm run build:styles:dev && npm run build:styles:prod",
+    "build:styles:dev": "cross-env NODE_ENV=production postcss src/styles/default.css -o styles/default.css --verbose",
+    "build:styles:prod": "cross-env NODE_ENV=production MINIFY_STYLES=true postcss src/styles/default.css -o styles/default.min.css --verbose",
     "test": "jest --detectOpenHandles",
     "test:watch": "jest --detectOpenHandles --watch",
-    "prepare": "npm run build:prod",
-    "prepack": "cp ../README.md ./README.md && cp ../LICENSE ./LICENSE",
+    "prepare": "npm run build:dev",
+    "prepack": "npm run build:prod && cp ../README.md ./README.md && cp ../LICENSE ./LICENSE",
     "postpack": "rm -rf ./README.md && rm -rf ./LICENSE",
```

Comment:
what is this for? why do we remove readme and license

Reply:
License and Readme are moved from root of project to library before publish, so by this library hasn't Readme in git, but we have it in npm https://www.npmjs.com/package/@asyncapi/react-component?activeTab=dependencies
After postpack we remove it (but these are copies).

Reply:
perfect, thanks for explanation, I wish there was `package.yml` so we could have comments for future :(

Reply:
🤦🏼 Due to this https://github.com/asyncapi/asyncapi-react/blob/master/package.json#L20 from this commit https://github.com/asyncapi/asyncapi-react/commit/a00fcc9cd98de3133397eac5d6a02ecb6163ebcd currently we haven't License and Readme on npm...

Reply:
I will fix it the PR with bumping dependencies for "old" component.